src/Core/Animation/Skinning/RotationCenterSkinning.cpp

```diff
@@ -1,3 +1,5 @@
+#include <Core/Mesh/TopologicalTriMesh/TopologicalMesh.hpp>
```

Comment:
I don't like this kind of fix, the order in which we include files should not cause compilation error.
Can you explain the rational behind this choice ?

If this due to the fact that we use a plugin to extend Eigen, that means that this plugin is not used everywhere, but only in files that include `TopologicalMesh.hpp`.
Why not defining the plugin using a preprocessor variable defined in cmake ? So we are sure **all** files in Radium use it ?

Reply:
I had problems trying to fix this.
Actually, the issue comes from the addition of the constructor `Matrix<T>(T value)` which fills all the matrix entries with the given value.
When defining classes with Eigen matrices as attributes, or functions with both Eigen matrix parameter and such class parameter signatures, then the compiler cannot decide which one to call.
For example it was the case there:
```c++
Grid<Scalar, 2> grid( Vector2(3,3) );
grid.at( {0,0} ) = Scalar( 1 );
```
there are 2 possible calls: one with `Grid<Scalar, 2>::IdxVector`, e.g. `Vector2ui`, one with `Grid<Scalar, 2>::Iterator` which stores a `Vector2ui` and a `uint`, and can now be initialized through `{0,0}`.
Same with this call:
```c++
Grid<uint,2> grid( {nb_rows, nb_cols} );
```
For which both `Grid( const IdxVector& size = IdxVector::Zero(), const T& val = T() )` and `Grid( const Grid<T, D>& other )` are now valid interpretations.


If you have an idea on how to properly address this, please go on.

Reply:
We asked for an update on OpenMesh side to replace the falty code.

Reply:
No need to add a constructor to initialize a Matrix<T> to a constant value.
there is a method, Constant(...), already existing in Eigen that do that : 
From the Eigen doc : 
```
Similarly, the static method Constant(value) sets all coefficients to value. 
If the size of the object needs to be specified, the additional arguments 
go before the value argument, as in MatrixXd::Constant(rows, cols, value).
```
See also reference : 
https://eigen.tuxfamily.org/dox/classEigen_1_1DenseBase.html#aed89b5cc6e3b7d9d5bd63aed245ccd6d

Reply:
deprecated. Closing comment.